HLeithner/joomla-cms
# Add prepared statements for plg_content_pagenavigation

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Use the plugin in all ways you can think of.

* Activate the page navigation feature
* create multiple articles in the same category
* check the frontend if you can navigate to the next article
* set the ordering in the menu and the global options, try different dates (create,published, modified)



### Expected result
Nothing changed.
